calico/getting-started/kubernetes/managed-public-cloud/eks.mdx

```diff
@@ -70,12 +70,18 @@ Before you get started, make sure you have downloaded and configured the [necess
    eksctl create cluster --name my-calico-cluster --without-nodegroup
    ```
 
-1. Since this cluster will use {{prodname}} for networking, you must delete the `aws-node` daemon set to disable AWS VPC networking for pods.
+1. Since this cluster will use {{prodname}} for networking, you must delete the `aws-node` daemonset to prevent it from writing conflicting CNI config to your nodes.
```

Comment:
```suggestion
1. Since this cluster will use {{prodname}} for networking, you must delete the `aws-node` daemonset to prevent it from writing a conflicting CNI config to your nodes.
```

Reply:
Not sure I agree with this one.  In my head "CNI config" is a collective noun, but not sure its worth arguing :)

Reply:
Yikes, that's way too cool for me. I'm reading config = configuration, which is the whole thing. All or nothing.

You're saying that this usage is collective, meaning we can write some config to the nodes, meaning we can write/rewrite _some discrete portion of the total configuration_ (more than 0 but less than all)?

Reply:
Yeah, the CNI config directory `/etc/cni/net.d/` on the host gets config files written to it by various CNI plugins.  

In this (error) case there will be 2 config files - one from aws-node, and one from calico-node.  Sadly, the tiebreaker for which one to use is alphabetical, which calico loses to aws.